pages/docs/key-repositories.en-US.mdx

```diff
@@ -86,16 +86,16 @@ This monorepo includes the following frontend modules:
 
 - [Active visits](https://github.com/openmrs/openmrs-esm-patient-management/tree/main/packages/esm-active-visits-app) - provides a tabular overview of checked in patients for the current day.
 - [Appointments](https://github.com/openmrs/openmrs-esm-patient-management/tree/main/packages/esm-appointments-app) - provides capabilities to create, edit and manage patient appointments.
-- [Service queues](https://github.com/openmrs/openmrs-esm-patient-management/tree/main/packages/esm-outpatient-app) - provides capabilities to create, manage and view queues in a location.
+- [Service queues](https://github.com/openmrs/openmrs-esm-patient-management/tree/main/packages/esm-service-queues-app) - provides capabilities to create, manage and view queues in a location.
 - [Patient search](https://github.com/openmrs/openmrs-esm-patient-management/tree/main/packages/esm-patient-search-app) - provides the ability to search for existing patients using simple and advanced search.
 - [Patient registration](https://github.com/openmrs/openmrs-esm-patient-management/tree/main/packages/esm-patient-registration-app) - provides the ability to register new patients and verify them through configurable external patient registries.
-- [Patient list](https://github.com/openmrs/openmrs-esm-patient-management/tree/main/packages/esm-patient-list-app) - provides a UI to create and manage patient lists.
+- [Patient list](https://github.com/openmrs/openmrs-esm-patient-management/tree/main/packages/esm-patient-list-management-app) - provides a UI to create and manage patient lists.
 
 ### Laboratory app
 
 This frontend module utilizes the following widgets to manage laboratory requests and queues in a user-friendly manner:
 
-- [Laboratory header](https://github.com/openmrs/openmrs-esm-laboratory-app/tree/main/src/header) - Displays the page header details, including the page name, icon, and other relevant information.
+- [Laboratory header](https://github.com/openmrs/openmrs-esm-laboratory-app/tree/main/src) - Displays the page header details, including the page name, icon, and other relevant information.
```

Comment:
This doesn't make any sense as the link. Please find the laboratory header.

Reply:
> This doesn't make any sense as the link. Please find the laboratory header.
@brandones  @denniskigen  can you check this is this correct


https://github.com/openmrs/openmrs-esm-laboratory-app/blob/main/src/components/overlay/overlay.component.tsx